havoc/single_line_grafana_annotator_test.go

```diff
@@ -0,0 +1,39 @@
+package havoc
+
+import (
+	"context"
+	"os"
+	"time"
+
+	"github.com/chaos-mesh/chaos-mesh/api/v1alpha1"
+	"github.com/rs/zerolog"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+func ExampleNewChaos() {
+	testLogger := zerolog.New(zerolog.ConsoleWriter{Out: os.Stderr}).With().Timestamp().Logger()
+	client := fake.NewFakeClient()
+	podChaos := &v1alpha1.PodChaos{ /* PodChaos spec */ }
+	chaos, err := NewChaos(ChaosOpts{
+		Object:      podChaos,
+		Description: "Pod failure example",
+		DelayCreate: 5 * time.Second,
+		Client:      client,
+		Logger:      &testLogger,
+	})
+	if err != nil {
+		panic(err)
+	}
+	logger := NewChaosLogger(testLogger)
+	annotator := NewSingleLineGrafanaAnnotator(
+		"http://grafana-instance.com",
+		"grafana-access-token",
+		"dashboard-uid",
+		Logger,
+	)
+	chaos.AddListener(logger)
+	chaos.AddListener(annotator)
+
+	chaos.Create(context.Background())
+	// Output:
```

Comment:
Is it urging me to write more code to handle output? Not sure what to do

Reply:
No, this is just how [Go examples verify output when run](https://go.dev/blog/examples) (my least favorite part of how it works). This is just saying that we don't expect any output from this code. I dunno if the chaos experiment would have some dummy logging we could print out?